fluent_comments/utils.py

```diff
@@ -26,14 +26,15 @@ def get_comment_template_name(comment):
     ]
 
 
-def get_comment_context_data(comment, action=None):
+def get_comment_context_data(comment, action=None, request=None):
```

Comment:
note that `get_comment_context_data` is called in 3 places, and each one should include the request.